draft-ietf-snac-simple.xml

```diff
@@ -206,6 +206,13 @@
 	<dt>RA Beacon:</dt>
 	<dd>A Router Advertisement (RA) that is multicast on a link so that hosts can see that the router is still present. This is in
 	  contrast to a unicast RA sent in response to the router solicit.</dd>
+	<dt>ULA Site Prefix:</dt>
+	<dd>A Unique Local Address /48 prefix <xref target="RFC4193"/> randomly generated by each stub router for use in providing
+	  ULA Link Prefixes to the stub network and the adjacent infrastructure link, as needed.</dd>
```

Comment:
providing should be allocating. "as needed" is probably not making this clearer.

Reply:
Updated.